packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -46,6 +41,27 @@ if ([production, development].indexOf(mode) < 0) {
     throw new Error(`Invalid mode "${mode}"`)
 }
 
+// for API convenience, add the leading slash if missing
+export const EXT_OVERRIDES_DIR =
+    typeof pkg?.ccExtensibility?.overridesDir === 'string' &&
+    !pkg?.ccExtensibility?.overridesDir?.startsWith('/')
+        ? '/' + pkg?.ccExtensibility?.overridesDir
+        : pkg?.ccExtensibility?.overridesDir
+export const EXT_EXTENDS = pkg?.ccExtensibility?.extends
+export const EXT_EXTENDABLE = pkg?.ccExtensibility?.extendable
```

Comment:
What is the definition of extendable? Is this Can you give me some examples of it's usage? Specifically since it's an array, what is an example of `extendable` when it has multiple values in it?

Reply:
It's an array in case someone makes an `extendable` project that needs to import and handle multiple base paths. But it doesn't have to be an array, that's a bit pre-emptive

Reply:
I'll just make it a string and be done, probably wrong for it to be an array

Reply:
Is `extendable` entirely preemptive? Do we need it for this initial implementation?

Reply:
We definitely need it for the initial implementation. Making it an array instead of a string is preemptive.

Reply:
For clarity, the alternative (non-option) to the `ccExtensibility.extenadable` would be hard-coding `retail-react-app` in the SDK. So, the value is required